src/reconcile/rec_write.c

```diff
@@ -910,49 +928,52 @@ __rec_init(WT_SESSION_IMPL *session,
 #endif
 
 	/*
+	 * When operating on the lookaside table, we can't save updates and
+	 * there is no point trying update/restore eviction: just evict pages
+	 * as soon as we can.
+	 */
+	if (F_ISSET(btree, WT_BTREE_LOOKASIDE))
+		LF_CLR(WT_REC_LOOKASIDE | WT_REC_UPDATE_RESTORE);
```

Comment:
I don't believe these flags can be set because our callers never set them (and `__evict_review()` explicitly checks). We could use an assert instead, so we don't imply it's a configuration we expect.